<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>100</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>ALM Firmware Updater v0.1</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="0">
       <widget class="QTextEdit" name="dfuResultsTextEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QPushButton" name="flashButton">
        <property name="text">
         <string>Flash</string>
        </property>
       </widget>
      </item>

      <item row="5" column="0">
	<widget class="QProgressBar" name="spinner">
       </widget>
      </item>


      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="fileBrowseLineEdit">
          <property name="readOnly">
           <bool>false</bool>
          </property>
          <property name="placeholderText">
           <string>Select Firmware File</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="fileBrowseButton">
          <property name="text">
           <string>Browse</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="listDevicesButton">
        <property name="text">
         <string>List Devices</string>
        </property>
       </widget>
       </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>fileBrowseButton</tabstop>
  <tabstop>flashButton</tabstop>
  <tabstop>fileBrowseLineEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
